src/main/java/org/spongepowered/common/mixin/api/mcp/world/raid/RaidMixin_API.java

```diff
@@ -0,0 +1,83 @@
+package org.spongepowered.common.mixin.api.mcp.world.raid;
+
+import java.util.ArrayList;
```

Comment:
These imports are odd, we have the format where `java.`/`javax.` are at the end in their own separate block.